QUEST_LV_0100.tsv

```diff
@@ -4198,176 +4198,176 @@ QUEST_LV_0100_20150317_004197	$Destroy the West Barrier in Nuotori Cliff
+QUEST_LV_0100_20150317_004252	$Protect Archivist Jonas
+QUEST_LV_0100_20150317_004253	$You were attacked by monsters while you were persuading Jonas. Protect Jonas from the monsters.
+QUEST_LV_0100_20150317_004254	$It seems that you defeated all the attacking monsters, and the Archivist Jonas regained his memory. Try persuading Jonas again to go back to camp with you and meet Niels.
+QUEST_LV_0100_20150317_004255	$Talk to Niels
+QUEST_LV_0100_20150317_004256	$You should retrieve the records which Jonas gave to Niels. Talk to Niels.
+QUEST_LV_0100_20150317_004257	$Talk to Merchant Davio
+QUEST_LV_0100_20150317_004258	$Talk to Davio and ask for a medicine that can regain Jonas' memory.
+QUEST_LV_0100_20150317_004259	$Bring the reports Jonas requested and find a way to feed the medicine to Jonas.
+QUEST_LV_0100_20150317_004260	$Defeat the monster that is disturbing Jonas's thoughts.
+QUEST_LV_0100_20150317_004261	$Archivist Jonas seem to have finally regained his memory. Jonas told you to follow him as there is a task that you are qualified to do. In order for Jonas to guide you safely to Beacon Hill Pass, defeat monsters along the way.
+QUEST_LV_0100_20150317_004262	$It seems that you have successfully arrived at the destination with Archivist Jonas. Talk to Jonas.
+QUEST_LV_0100_20150317_004263	$Talk to Gorath
+QUEST_LV_0100_20150317_004264	$You made Badat to return safely. Talk to Gorath about it.
+QUEST_LV_0100_20150317_004265	$Defeat Molek
+QUEST_LV_0100_20150317_004266	$We don't know where Molek chased after us from, but let's defeat it first!
+QUEST_LV_0100_20150317_004267	$You have defeated Molek. Talk to Gorath again.
+QUEST_LV_0100_20150317_004268	$Defeated Molek
+QUEST_LV_0100_20150317_004269	$It seems that Jonas' memories are coming back to him. Talk to him.
+QUEST_LV_0100_20150317_004270	$Observe the Eyes of the Great King besides the Archivist Jonas.
+QUEST_LV_0100_20150317_004271	$Archivist Jonas is asking you to prove that you are qualified for his task by using the Eyes of the Great King.
+QUEST_LV_0100_20150317_004272	$The Eyes of the Great King have responded. Talk to Jonas.
+QUEST_LV_0100_20150317_004273	$Talk to the Medicine Dealer
+QUEST_LV_0100_20150317_004274	$Merchant Davio told you that he needs one more herb to complete the medicine, but he already sent off the Medicine Dealer to get the herb. Go meet the Medicine Dealer at Kapas Highway.
+QUEST_LV_0100_20150317_004275	$Collect Canyon Metos
+QUEST_LV_0100_20150317_004276	$The Medicine Dealer told you that all other ingredients is ready, except for Canyon Metos. Collect Canyon Metoses in Metos Field in order to complete the medicine that will bring back Jonas' memories.
+QUEST_LV_0100_20150317_004277	$You have enough Canyon Metos. Go back to the Medicine Dealer and complete the medicine that will bring back Jonas' memories.
+QUEST_LV_0100_20150317_004278	$Niels asked Merchant Davio for a medicine that can bring back Jonas' memories. Retrieve the medicine from Davio.
```

Comment:
'memory' is wrong. 정신 is a broad term, so it's hard to pinpoint since I don't know the story about this quest. But 'consciousness' is close to it. However, SHU has informed me a little bit about the background.

What about "mind"?